docs/reference/elasticsearch/configuration-reference/auding-settings.md

```diff
@@ -20,7 +20,7 @@ If configured, auditing settings must be set on every node in the cluster. Stati
 
 $$$xpack-security-audit-enabled$$$
 
-`xpack.security.audit.enabled`
+`xpack.security.audit.enabled` ![logo cloud](https://doc-icons.s3.us-east-2.amazonaws.com/logo_cloud.svg "Supported on {{ess}}")
```

Comment:
we don't really use "Elasticsearch service" as a term anymore. would prefer to specify "Elastic Cloud Hosted and and Elastic Cloud Enterprise" - unfortunately we can't say "Elastic Cloud" because that implies serverless and kubernetes :(

Reply:
also: have we confirmed that ECE and ECH have the same supported settings? I wasn't sure if there was a discrepancy (as far as I knew offhand, ECE didn't "block" settings like ECH did)